src/shared/ui/inputs/Input/Input.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Input } from "./Input";
+
+const meta = {
+  args: {
+    className:
+      "block min-h-control-size-base w-full rounded-border-radius-base border border-essential-subdued bg-background-base py-spacing-tighter-2 font-semibold shadow-essential-base placeholder:text-color-subdued hover:border-essential-base focus:border-essential-base focus:border-white focus:ring-white",
```

Comment:
If feels to be a lot of classes the consumer needs to pass each time he wants to use an input. Do you think we can provide some good defaults to the `<Input />`? Maybe all input have a border? or a `font-semibold`? or a `placeholder:text-color-subdued`?

(I don't know really ^^. We should check different usage in the Spotify app and extract some good defaults if there are some 👍)

Reply:
From what I can see on the site, there are two types of input, and the one with the focus on the white borders comes up most often.

I think it's ok that this is the default style for input, let me know what you think!